docs/modules/ROOT/pages/python-client-with-pandas-and-DBAPI.adoc

```diff
@@ -0,0 +1,303 @@
+used to store JSON formatted strings in Hazelcast cluster.
+Now we can get the data from Hazelcast cluster and analyze it with Pandas.
+
+[source, python]
+----
+import hazelcast
+from hazelcast.core import HazelcastJsonValue
+from matplotlib import pyplot as plt
+import numpy as np
+import pandas as pd
+
+# Create Hazelcast client
+client = hazelcast.HazelcastClient()
+
+# Get an IMap for storing patient's DataFrame objects
+patients = client.get_map("patients").blocking()
+
+# Store the blood pressure and heart rate data of fifty patients in a DataFrame and load it to Hazelcast cluster as JSON
+for pid in range(0, 50):
+    # Create DataFrame with random values
+    df = pd.DataFrame(
+        data={
+            "blood_pressure": np.random.randint(80, 120, size=(75,)),
+            "heart_rate": np.random.randint(60, 100, size=(75,)),
+        },
+        index=pd.date_range("2023-01-15", periods=75, freq="H"),
+    )
+    # Load DataFrame to Hazelcast cluster as HazelcastJsonValue
+    patients.put(pid, HazelcastJsonValue(df.to_json()))
+
+pid = np.random.randint(0, 50)
+# Retrieve the data of a random patient
+df = pd.read_json(patients.get(pid).to_string())
+
+# Plot the data
+df.plot(use_index=True, y=["blood_pressure", "heart_rate"], figsize=(15, 5), kind="line")
+plt.title(f"Blood Pressure and Heart Rate Plot of Patient-{pid}")
+plt.show()
+client.shutdown()
+----
+
+In this example, we have created a map named `patients` and stored the blood pressure and heart rate data of fifty patients in a DataFrame and load it to Hazelcast cluster as JSON.
+Then, we have retrieved the data of a random patient and plotted it.
+
+== Using DPABI API with Pandas
+Hazelcast have a https://hazelcast.readthedocs.io/en/stable/api/db.html[DBAPI] which is compatible with https://pandas.pydata.org/docs/reference/index.html#api[Pandas]. You can use this API to analyze your data in Hazelcast cluster with Pandas.
+For DBAPI, we are going to use another example. So, do not forget that these two are completely different examples.
+Let's create a map, we have given a different name to the map name send to cluster, since if you are using Hazelcast Viridian as cluster
+we do not want to override the data to the map that we have created in the previous example.
+
+[source, python]
+----
+import hazelcast
+from hazelcast.db import connect
+from hazelcast.core import HazelcastJsonValue
+import pandas as pd
+import numpy as np
+from matplotlib import pyplot as plt
+
+client = hazelcast.HazelcastClient()
+
+# Create a Distributed Map in the cluster
+patients = client.get_map("patientsWithDBAPI").blocking()
+----
+
+Now, we are going to create Connection object instance and a cursor object instance. We will use these instances to execute our SQL queries.
+
+[source, python]
+----
+# Create a connection object
+conn = connect()
+cursor = conn.cursor()
+----
+
+Now, we are ready to execute our queries. Let's create a table named `patients` first.
```

Comment:
You used the name `patientsWithDBAPI` above, but now creating the mapping for `patients`. Is there a reason?